pages/quantum.js

```diff
@@ -42,145 +43,417 @@ import {
-            title: "twitter title of the purpose of this icon",
-          },
-          {
-            name: "youtube",
-            url: "http://youtube.com",
-            title: "youtube title of the purpose of this icon",
-          },
-        ]}
-      />
-      <Accordion
-        items={[
-          {
-            title: "Test 1",
-            content: "Content Text...",
-          },
-          {
-            title: "Test 2",
-            content: "Content Text...",
-          },
-        ]}
-      />
-      <Popover trigger="Popover Trigger" onClose={() => {}}>
-        Popover test
-      </Popover>
-      <Tooltip placement="top" text="Tooltip Text">
-        Tooltip Trigger
-      </Tooltip>
-      <CheckboxGroup
-        options={[{ checked: false, label: "Test", name: "Test" }]}
-      />
-      <RadioGroup
-        name="test"
-        options={[
-          {
-            value: "Test",
-            label: "Test",
-          },
-        ]}
-      />
-      <Toggle id="right" />
-      <Form><input /></Form>
-      <Carousel
-        cards={[
-          {
-            imagePath: "https://assets.catho.com.br/logo/svg/blue.svg",
-            imageDescription: "test",
-            title: "title",
-            description: "description...",
-          },
-        ]}
-      />
-      <SegmentedControl
-        name="items-text"
-        items={[
-          {
-            label: "Test",
-            value: "Test ",
-            checked: true,
-          },
-        ]}
-      />
-      <ControlledModal/>
-      <ControlledSnackBar/>
+      <div class="MarginTopBottom ColoredBackground PaddingTopBotttom">
+        <h4 class="TextFormattingH4 PaddingLeftRight">Breadcrumbs</h4>
+        <h6 class="TextFormattingH6 PaddingLeftRight">
+          Breadcrumbs creates a trail of links to help the user to locate the
+          current page within the website's hierarchy.
+        </h6>
+
+        <Breadcrumbs items={breadcrumbsMockItems} />
+
+        <div class="MarginTopBottom PaddingLeftRight">
+          <h4 class="TextFormattingH4">TabbedView</h4>
+          <h6 class="TextFormattingH6">
+            TabbedView and Tab are components created to organize content into
+            separate views where only one view can be visible at a time.
+          </h6>
+          <TabbedView activeTab="TabTest">
+            <Tab title="TabTest">Tab with TabbedView</Tab>
+            <Tab title="TabTest 2">Second tab with TabbedView</Tab>
+            <Tab title="TabTest 3">Third tab with TabbedView</Tab>
+          </TabbedView>
+        </div>
+      </div>
+
+      <div class="FlexContainer MarginTopBottom ColoredBackground PaddingTopBotttom">
+        <div class="WhiteBackground PaddingLeftRight RoundedEdges MarginLeftRight">
+          <div>
+            <h4 class="TextFormattingH4">Rows and Colums</h4>
+          </div>
+          <Row>
+            <Col>
+              <Hide>Hide</Hide>
+            </Col>
+          </Row>
```

Comment:
What do you think about putting some more "row" and "col" with content to make examples more realistic?

Reply:
Hide could be a separate example describing how it works, what do you think about it?